### PR TITLE
test: Adding python unit tests

### PR DESCRIPTION
**Pull request by rakatyal**
[DI-2453](https://issues.corp.twilio.com/browse/DI-2453)

### CODE REVIEW

File: examples/python/Dockerfile [hunk 1]
```diff
@@ -5,17 +5,19 @@ ENV PYTHONBUFFERED 1
...
 RUN git clone https://github.com/twilio/twilio-python.git
 WORKDIR /app/twilio-python/
+#TODO: Change to main when GA
 RUN git fetch && git checkout 8.0.0-rc && git pull
 
-RUN rm -rf twilio/rest tests
+RUN rm -rf tests/ twilio/rest/
 
-COPY twilio/rest twilio/rest
+COPY twilio twilio
```
Comment (claudiachua) [inline examples/python/Dockerfile:13]: Should this be copying twilio/rest? Since only twilio/rest is being removed in the previous line

Comment (rakatyal) [reply examples/python/Dockerfile:13]: Think it works either way. Wanted to keep it consistent with [node](https://github.com/twilio/twilio-oai-generator/blob/main/examples/node/Dockerfile#L9)

File: examples/python/tests/unit/account_test.py [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+        mock_request.assert_called_once_with("POST", "https://api.twilio.com/2010-04-01/Accounts.json", **request_args)
+
+    def test_account_shortcut(self, mock_request):
+        mock_request.return_value = Response(201, "{\"sid\": \"123\"}")
+
+        self.client.api.V2010.account.calls.create(required_string_property="phone home", test_method="post")
+        request_args = self.generic_request_args
+        request_args["data"] = {"RequiredStringProperty": "phone home", "TestMethod": "post"}
+
+        mock_request.assert_called_once_with("POST", f"https://api.twilio.com/2010-04-01/Accounts/{self.sid}/Calls.json", **request_args)
```
Comment (Hunga1) [inline examples/python/tests/unit/account_test.py:30]: What is this test case testing exactly? I'm having difficulty seeing a difference between this test and the test_create_call test in examples/python/tests/unit/account/call_test.py. Is it the sid value used to initialize the client inserted into the endpoint URL automatically?

**Review by claudiachua — APPROVED** [review]

File: examples/python/tests/unit/account_test.py (hunk 2 above)
Comment (rakatyal) [reply examples/python/tests/unit/account_test.py:30]: That is correct. In the account/call_test user provides the account sid and in this case it does not. Here we use the sid used to [initialize the client](https://github.com/twilio/twilio-oai-generator/blob/main/examples/python/twilio/rest/api/v2010/account/__init__.py#L342)

**Pull request merged**